cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -552,7 +552,17 @@ find_package(Python3 3.8 COMPONENTS ${python_components})
 #---Check for OpenGL installation-------------------------------------------------------
 if(opengl)
   message(STATUS "Looking for OpenGL")
-  find_package(OpenGL)
+  if(APPLE)
+    if(cocoa)
+      set(CMAKE_FIND_FRAMEWORK FIRST)
+    endif()
+    find_package(OpenGL)
+    if(cocoa)
+      set(CMAKE_FIND_FRAMEWORK LAST)
+    endif()
+  else()
+    find_package(OpenGL)
+  endif()
```

Comment:
```suggestion
  if(APPLE and cocoa)
      set(CMAKE_FIND_FRAMEWORK FIRST)
      find_package(OpenGL)
      set(CMAKE_FIND_FRAMEWORK LAST)
  else()
    find_package(OpenGL)
  endif()
```

Reply:
The flag has only an effect on macOS anyway, so why not just:
```cmake
  if(cocoa)
      set(CMAKE_FIND_FRAMEWORK FIRST)
      find_package(OpenGL)
      set(CMAKE_FIND_FRAMEWORK LAST)
  else()
    find_package(OpenGL)
  endif()
```
But that raises the question why this depends on cocoa? Would it not be okay to just do:
```cmake
    set(CMAKE_FIND_FRAMEWORK FIRST)
    find_package(OpenGL)
    set(CMAKE_FIND_FRAMEWORK LAST)
```